modules/rollback-operator-upgrades-cli.adoc

```diff
@@ -0,0 +1,166 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-operator.adoc
+:_content-type: PROCEDURE
+[id="rollback-operator-upgrades_{context}"]
+= Rolling back an Operator upgrade by using the CLI
+
+You can roll back the Operator version by using CLI commands.
+
+.Procedure
+
+. Delete the {olm} subscription by running the following command:
+* For {ocp}:
++
+[source,terminal]
+----
+$ oc -n rhacs-operator delete subscription rhacs-operator
+----
+* For Kubernetes:
++
+[source,terminal]
+----
+$ kubectl -n rhacs-operator delete subscription rhacs-operator
+----
+. Delete the cluster service version (CSV) by running the following command:
+* For {ocp}:
++
+[source,terminal]
+----
+$ oc -n rhacs-operator delete csv -l operators.coreos.com/rhacs-operator.rhacs-operator
+----
+* For Kubernetes:
++
+[source,terminal]
+----
+$ kubectl -n rhacs-operator delete csv -l operators.coreos.com/rhacs-operator.rhacs-operator
+----
+. Determine the previous version you want to roll back to by choosing one of the following options:
+[NOTE]
+====
+This procedure can only be used for {product-title-short} release 3.74 and earlier when the `rocksdb` database is installed.
```

Comment:
This note should be under the second path where the user creates a pod.  This first path (where they query the Central API) is okay for both postgres and rocksdb.